.golangci.yml

```diff
@@ -30,7 +30,7 @@ linters:
     - gocognit
     - gofmt
     # - gomnd
-    - gomoddirectives
+    # - gomoddirectives
```

Comment:
why uncomment

Reply:
Otherwise `replace` would not be allowed